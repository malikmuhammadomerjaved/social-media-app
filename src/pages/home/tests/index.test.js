import { render } from "@testing-library/react";
import Home from "../index";

describe("<Home />", () => {
  const { container } = render(<Home />);

  it("should match Home component to snapshot", () => {
    expect(container).toMatchSnapshot();
  });
});
