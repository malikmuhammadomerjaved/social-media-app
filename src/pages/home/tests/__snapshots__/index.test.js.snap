// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should match Home component to snapshot 1`] = `
<div>
  <main
    class="home-main-container"
  >
    <figure
      class="landing-page-image-container"
    >
      <img
        alt="Events"
        class="main-image"
        src="/images/landing-page/event-image.png"
      />
    </figure>
    <article
      class="home-content-wrapper"
    >
      <article
        class="main-heading-wrapper"
      >
        <h1
          class="plain-heading"
        >
          Imagine if
        </h1>
        <h1
          class="plain-heading gradient"
        >
          Snapchat
        </h1>
        <h1
          class="plain-heading"
        >
          had events.
        </h1>
      </article>
      <h2
        class="main-subheading"
      >
        Easily host and share events with your friends across any social media.
      </h2>
    </article>
    <article
      class="common-button-wrapper"
    >
      <button
        class="common-button"
      >
        ðŸŽ‰ Create my event
      </button>
    </article>
  </main>
</div>
`;
